echo_handlers.go

```diff
@@ -198,18 +202,17 @@ func (svc *Service) AppsShowHandler(c echo.Context) error {
 		renewsIn = getEndOfBudgetString(endOfBudget)
 	}
 
-	return c.Render(http.StatusOK, "apps/show.html", map[string]interface{}{
-		"App":                   app,
-		"PaySpecificPermission": paySpecificPermission,
-		"RequestMethods":        requestMethods,
-		"ExpiresAt":             expiresAt,
-		"ExpiresAtFormatted":    expiresAtFormatted,
-		"User":                  user,
-		"LastEvent":             lastEvent,
-		"EventsCount":           eventsCount,
-		"BudgetUsage":           budgetUsage,
-		"RenewsIn":              renewsIn,
-		"Csrf":                  csrf,
+	return c.JSON(http.StatusOK, ShowAppResponse{
+		App:                   app,
+		PaySpecificPermission: paySpecificPermission,
+		RequestMethods:        requestMethods,
+		ExpiresAt:             expiresAt.Unix(),
+		ExpiresAtFormatted:    expiresAtFormatted,
+		LastEvent:             lastEvent,
+		EventsCount:           eventsCount,
+		BudgetUsage:           budgetUsage,
+		RenewsIn:              renewsIn,
+		Csrf:                  csrf,
```

Comment:
I think it is weird that we return Csrf like this manually. Isn't there a middleware or something? and why do we even need it in this response?